frontend/src/metabase/components/MetadataInfo/MetadataInfo.styled.jsx

```diff
@@ -28,13 +28,14 @@ export const LabelContainer = styled.div`
   display: inline-flex;
   align-items: center;
   column-gap: ${space(0)};
-  font-size: 12px;
+  font-size: 12.8px;
```

Comment:
It is a very unique value for the `font-size` in our codebase that makes me question if we really need exact this value instead of close but consistently used in our places values

Reply:
yeah I agree with this and my original intention with this `*Info` components was to make them scale in size nicely so that they can be used in a variety of ways. So, I've tweaked the implementation and now all sub-components use `em` and the consuming component, `DimensionInfoPopover`, is that one that decides the size -- so now there's a `font-size: 14px` set at the top level